pkg/tsdb/influxdb/fsql/fsql_test.go

```diff
@@ -109,3 +118,14 @@ func mustQueryJSON(t *testing.T, refID, sql string) []byte {
 	}
 	return b
 }
+
+func freeport() (port int, addr string) {
+	l, err := net.ListenTCP("tcp", &net.TCPAddr{IP: net.ParseIP("127.0.0.1")})
+	if err != nil {
+		return 0, "err"
+	}
+	defer l.Close()
+	a := l.Addr().(*net.TCPAddr)
+	port = a.Port
+	return port, a.String()
```

Comment:
not necessary to return the port integer, so this could be simplified:

```suggestion
func freeport() string {
	l, err := net.ListenTCP("tcp", &net.TCPAddr{IP: net.ParseIP("127.0.0.1")})
	if err != nil {
		return "err"
	}
	defer l.Close()
	a := l.Addr().(*net.TCPAddr)
	return a.String()
```